di_website/templates/ourteam/team_member_page.html

```diff
@@ -16,22 +14,29 @@
     <div class="row row--narrow">
         <div class="hero__profile">
             {% if self.image %}
-              {% image self.image fill-260x260-c100 as img %}
-              <img src="{{img.url}}" alt="{{self.name}} photograph">
+            {% image self.image fill-260x260-c100 as img %}
+            <img src="{{img.url}}" alt="{{self.name}} photograph">
```

Comment:
Can we quickly add `width=260px height=260px` here to see if it lets us scale up smaller photos?